src/clj/game/cards/programs.clj

```diff
@@ -144,6 +144,26 @@
                    :msg (msg "install " (:title target))
                    :effect (req (when (can-pay? state side nil :credit (:cost target))
                                   (runner-install state side target)))}]})
+   "Consume"
+   {:events {:runner-trash {:req (req (= (:side target) "Corp"))
+                              :optional
+                              {:prompt "Place a virus counter on Consume?"
+                               :yes-ability
+                               {:effect (effect (add-counter :runner card :virus 1)
+                                                (system-msg :runner (str "places 1 virus counter on Consume")))}}}}
+    :abilities [{:cost [:click 1]
+                 :effect (req (gain state side :credit (* 2 (get-virus-counters state side card)))
+                              (update! state side (assoc-in card [:counter :virus] 0))
+                         (when-let [hiveminds (filter #(= "Hivemind" (:title %)) (all-active-installed state :runner))]
```

Comment:
This `(when-let [...] ...)` clause should be aligned with `(update! ...)` above.